Core/src/IfcGeometryConverter/GeometrySettings.h

```diff
@@ -164,6 +164,44 @@ namespace OpenInfraPlatform
 					}
 				}
 
+				bool areEqual(const double first,const double second) const
+				{
+					if (abs(first - second) < getPrecision()) 
+					{
+						return true;
+					}
+					return false;
+				}
+
+				bool areEqual(const carve::geom::vector<2>& first, const  carve::geom::vector<2>& second) const
+				{
+					if (abs(first.x - second.x) < getPrecision() && abs(first.y - second.y) < getPrecision())

```

Comment:
Please rather use the call to `areEqual` with doubles.

Reply:
Agree, thank you for the hint

Reply:
Resolved: 1b84b1d